stats/stats/src/charts/lines/average_txn_fee.rs

```diff
@@ -33,7 +36,7 @@ const ETHER: i64 = i64::pow(10, 18);
 pub struct AverageTxnFeeStatement;
 
 impl StatementFromRange for AverageTxnFeeStatement {
-    fn get_statement(range: Option<Range<DateTimeUtc>>) -> Statement {
+    fn get_statement(range: Option<Range<DateTimeUtc>>, _: &BlockscoutMigrations) -> Statement {
```

Comment:
Similar to `AverageGasPriceStatement` considerations apply

Reply:
1233.900141548s -> ~17s
definitely better